tests/risks/test_effect.py

```diff
@@ -413,6 +414,95 @@
 #                                                   source=lambda index: pd.Series(0.1, index=index))
 #
 #     assert np.allclose(from_yearly(0.1, time_step)*50, em(simulation.get_population().index))
+def _setup_risk_effect_simulation(
```

Comment:
this is the same as the old _setup_risk_simulation except the user passes in a risk_effect that gets used as a component in the sim rather than defaulting to NonLogLinearRiskeffect